package.json

```diff
@@ -10,8 +10,14 @@
   "scripts": {},
   "author": "Pabs",
   "dependencies": {
-    "angular2": "2.0.0-alpha.44",
-    "systemjs": "^0.19.6"
+    "angular2": "^2.0.0-alpha.52",
+    "systemjs": "^0.19.6",
+
+    "es6-promise": "^3.0.2",
+    "es6-shim": "^0.33.3",
+    "reflect-metadata": "0.1.2",
+    "rxjs": "5.0.0-alpha.14",
+    "zone.js": "0.5.8"
```

Comment:
Reason for this dependencies: https://github.com/angular/angular/commit/4e16feaf722e8b1b3b14eff83b3fe5d2635daa19